### PR TITLE
=plugin migrate FastClasspathScanner to ClassGraph

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

File: project/ParadoxSupport.scala [hunk 1]
```diff
@@ -24,8 +22,13 @@ object ParadoxSupport {
...
       val classloader = new java.net.URLClassLoader(classpath, this.getClass().getClassLoader())
-      lazy val scanner = new FastClasspathScanner("akka").addClassLoader(classloader).scan()
-      val allClasses = scanner.getNamesOfAllClasses.asScala.toVector
+      import _root_.io.github.classgraph.ClassGraph
+      lazy val scanner = new ClassGraph()
+        .whitelistPackages("akka")
+        .verbose()
+        .addClassLoader(classloader)
+        .scan()
+      val allClasses = scanner.getAllClasses.getNames.asScala.toVector
```
Comment (patriknw) [inline project/ParadoxSupport.scala]: is it the same thing that has moved, or what is the background?

Comment (hepin1989) [reply project/ParadoxSupport.scala]: The FastClassPathScananer is replaced by the ClassGraph, and only ClassGraph is being maintained now

**Review by patriknw — APPROVED** [review]
LGTM

**Review by chbatey — APPROVED** [review]
LGTM, thanks

**Comment by patriknw** [comment]
@hepin1989 Now it prints many many
```
2018-10-17T08:33:46.357+0200	ClassGraph	------ Skipping non-whitelisted path: scala/reflect/internal/Importers$$anon$3.class
```

when running `sbt akka-docs/paradox`. Can we silence that?

**Comment by patriknw** [comment]
it also seems much slower than before (perhaps due to the logging?)
`akka-docs/paradox` takes ages, and that matters when writing documentation

**Comment by raboof** [comment]
For me it went from an already-painful 10s to ~14s... would be great if we could speed this up somehow.

**Comment by patriknw** [comment]
The problem was `verbose()`. Removed in https://github.com/akka/akka/pull/25806

**Pull request merged**